viewer-prototype/src/common/experiment-manager.ts

```diff
@@ -87,11 +88,32 @@ export class ExperimentManager {
             'traces': traceURIs
         }));
         const experiment = experimentResponse.getModel()
-        if (experiment && (experimentResponse.isOk() || experimentResponse.getStatusCode() === 409)) {
+        if (experiment && experimentResponse.isOk()) {
             this.addExperiment(experiment);
             this.experimentOpenedEmitter.fire(experiment);
             return experiment;
+        } else if (experiment && experimentResponse.getStatusCode() === 409) {
+            // Repost with a suffix as long as there are conflicts
+            let handleConflict = async function(tspClient: TspClient, tryNb: number): Promise<TspClientResponse<Experiment>> {
+                let suffix = '(' + tryNb + ')';
+                return await tspClient.createExperiment(new Query({
+                    'name': name + suffix,
+                    'traces': traceURIs
+                }))
+            }
+            let conflictResolutionResponse = experimentResponse;
+            let i = 1;
+            while (conflictResolutionResponse.getStatusCode() === 409) {
+                conflictResolutionResponse = await handleConflict(this.tspClient, i);
```

Comment:
forgot i++

Reply:
Ahh I missed that. I only tried having one conflict. I'm waiting for an update.